app/controllers/api/v1/captains_controller.rb

```diff
@@ -22,8 +22,13 @@ def create
       password: params[:password],
       password_confirmation: params[:password_confirmation]
     )
-    if params[:availability]
```

Comment:
To keep with the code style we've established, shouldn't this be in the validate_params function?

Reply:
Should what be in the validate_params? What you're commenting on is a removed line.